okio/src/commonTest/kotlin/okio/PathTest.kt

```diff
@@ -146,6 +146,39 @@ class PathTest {
     assertFalse(path.isRoot)
   }
 
+  @Test
+  fun windowsAbsolutePathWithVolumeLetterForwardSlashes() {
+    val path = "C:/Windows/notepad.exe".toPath()
+    assertEquals("C:/Windows/notepad.exe", path.toString())
+    assertEquals("C:/Windows".toPath(), path.parent)
+    assertEquals('C', path.volumeLetter)
+    assertEquals("notepad.exe", path.name)
+    assertTrue(path.isAbsolute)
+    assertFalse(path.isRoot)
+  }
+
+  @Test
+  fun windowsAbsolutePathWithVolumeLetterMixedSlashesBackslashFirst() {
+    val path = "C:\\Windows/notepad.exe".toPath()
+    assertEquals("C:\\Windows\\notepad.exe", path.toString())
+    assertEquals("C:\\Windows".toPath(), path.parent)
+    assertEquals('C', path.volumeLetter)
+    assertEquals("notepad.exe", path.name)
+    assertTrue(path.isAbsolute)
+    assertFalse(path.isRoot)
+  }
+
+  @Test
+  fun windowsAbsolutePathWithVolumeLetterMixedSlashesSlashFirst() {
+    val path = "C:/Windows\\notepad.exe".toPath()
+    assertEquals("C:/Windows/notepad.exe", path.toString())
```

Comment:
Paths separators are currently normalized based on the first slash in the path. This isn't necessarily wrong, but it is different from how JVM and most other systems behave.